src/bootstrap/config.go

```diff
@@ -91,6 +94,14 @@ func (config *Config) loadServerConfig() {
 	err = decoder.Decode(&config)
 	failOnError(err, "Error decoding JSON config file.")
 
+	if !strings.HasPrefix(config.SettingsFile, "/") {
```

Comment:
You can use the IsAbs function to check if the provided path is absolute: https://golang.org/pkg/path/filepath/#IsAbs

Reply:
Good callout!  Sorry, I don't often write in Go and my terrible C habits are showing.  Fixed.

Reply:
No worries :) The builtin path libs help keep things working for Windows paths as well.